compiler/fir/analysis-tests/testData/resolve/inlineClasses/inlineClassMembers.kt

```diff
@@ -0,0 +1,32 @@
+value class BackingFields(val x: Int) {
+    <!PROPERTY_WITH_BACKING_FIELD_INSIDE_INLINE_CLASS!>val y<!> = 0
+    var z: String
+        get() = ""
+        set(value) {}
+}
+
+class Val {
+    operator fun getValue(thisRef: Any?, kProp: Any?) = 1
+}
+inline class DelegatedProp(val x: Int) {
+    val testVal by <!DELEGATED_PROPERTY_INSIDE_INLINE_CLASS!>Val()<!>
+}
+
+inline class ReversedMembers(val x: Int) {
+    <!RESERVED_MEMBER_INSIDE_INLINE_CLASS{LT}!>fun <!RESERVED_MEMBER_INSIDE_INLINE_CLASS{PSI}!>box<!>() {}<!>
```

Comment:
DECLARATION_NAME light tree positioning strategy works erroneously here. I will fix this problem myself.